src/main/treeDataProviders/dependenciesTree/dependenciesRoot/rootTree.ts

```diff
@@ -89,23 +89,31 @@ export class RootNode extends DependenciesTreeNode {
     private static collectPaths(vulnerableDependencyId: string, children: DependenciesTreeNode[], size: number): IImpactGraph {
         let impactPaths: IImpactGraphNode[] = [];
         for (let child of children) {
+            if (size === RootNode.IMPACT_PATHS_LIMIT) {
+                break;
+            }
```

Comment:
I'm not sure about that, but I'm afraid that doing it might cause direct dependencies to appear as direct in some cases (the same issue we were afraid of when I worked on it in the IntelliJ plugin).
Is there a chance that a direct child will be skipped here, and the dependency will be marked as indirect? Or do we have a special logic for this scenario somewhere else?